maturity-model.md

```diff
@@ -0,0 +1,175 @@
+
+**Leadership**
+
+* LS-0:	Command & control culture, within a highly hierarchical organization.
+* LS-1: Some leaders are open to receiving feedback and creating an environment where people feel safe providing it.
+* LS-2:	Most leaders in the organization are open to receiving feedback and creating an environment where people feel safe providing it. Leaders show passivity about understanding whether all members feel empowered and enabled to share. Organization encourages leaders to actively seek voices not present in dialog out for inclusion.
+* LS-3: Members feel empowered and enabled to share opinions constructively on any matter relevant to their work or about which they feel passionate.
+
+**Organizational and Functional Structure**
+
+* OF-0:	Working groups tend to be static in terms of membership and skill sets.
+* OF-1:	Cross-functional teams exist, but team roles are often unclear and governance structures are vague.
+* OF-2:	Cross-functional teams are common, and teams post their roles and goals publicly.
+* OF-3:	Cross-functional teams are common and make their activities known broadly to the organization; in turn, the organization promotes best practices for working together.
+
+**Contribution**
+
+* CB-0:	Completly siloed, no collaboration outside the teams. Just some collaborations due to cross-functional teams.
+* CB-1:	Members of the organization and teams collaborate but frequently say it's "too difficult". Teams infrequently revisit the outcomes of their collaborations.
+* CB-2:	Members of the organization and teams actively seek opportunities to collaborate. Teams routinely discuss, revisit and debate the outcomes of their collaborative efforts, and make these outcomes available by default.
+* CB-3:	Members of the organization collaborate both internally and externally in ways that benefit all involved. Teams routinely discuss, revisit and debate the outcomes of their collaborative efforts, and share their learnings outside the organization, and make these outcomes externally available by default.
+
+### Community
+
+**Sharing Policies**
+
+* SP-0: No sharing culture nor written policies.
+* SP-1:	Some members of the organization unite to define values and principles, but are not clearly supported when they do.
+* SP-2:	Members of the organization collectively document shared visions and agreements like mission statements and codes of conduct, make them easily accessible, and reference them often. Onboarding materials and orientation rituals provide adequate context for helping new members understand how the organization will benefit from their contributions.
+* SP-3:	Shared values and principles inform decision-making, conflict resolution, and assessment processes among members of the organization, who reference these values and principles consistently in both verbal and written formats.
+
+**Feel part of the Organization**
```

Comment:
The points below sound fairly vague and like they apply to any organisation - not only to those trying to adopt InnerSource. I'd likely remove them or try to make them more relevant to the InnerSource context.

Reply:
Having a sense of ownership to a certain project, and increasing the engagement within the organization is one of the points of having an InnerSource initiative in place.

We tried to use this concept of 'feeling part of something', but happy to reword this to really include the concept of engagement. Any suggestion?

Reply:
Hmm - you explaining that increasing engagement can be one reason for introducing an InnerSource initiative helped me understand better, why this is part of the capabilities model. Do you think adding this explanation would make sense?